pkgs/by-name/wi/wireguard-go/package.nix

```diff
@@ -6,30 +6,26 @@
   wireguard-go,
 }:
 
+let
+  version = "0.0.20250522";
```

Comment:
I don't see any particular reason to have this in a let binding.

Reply:
> I don't see any particular reason to have this in a let binding.

Well, it is used twice - in the derivation as well as in the download URL. And I read in the style guide that simple let-binding are to be preferred over "rec { ... }" and similar constructions.

Which alternative would you prefer? Reverting to the finalAttrs pattern?

Reply:
Ok, so I just reverted to the finalAttrs pattern.